_api-reference/cat/cat-thread-pool.md

```diff
@@ -35,7 +34,7 @@ GET _cat/thread_pool
 
 All cat thread pool URL parameters are optional.
 
-In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/opensearch/rest-api/cat/index), you can specify the following parameter:
+In addition to the [common URL parameters]({{site.url}}{{site.baseurl}}/api-reference/cat/index), you can specify the following parameter:
```

Comment:
`[Compact and aligned text (CAT) API]({{site.url}}{{site.baseurl}}/api-reference/cat/index)`